src/pages/EditorTeam/EditorTeam.tsx

```diff
@@ -0,0 +1,300 @@
+  );
+};
+
+const EditorTeam: FC = () => {
+  const registry = useSelector((state: RootState) => state.income.registry);
+  const [rows, setRows] = useState<IPerson[] | undefined>();
+  const [team, setTeam] = useState<string>('');
+  const [previous, setPrevious] = useState<any>({});
+  const classes = useStyles();
+  const [openNewMember, setOpenNewMember] = useState<boolean>(false);
+  const [actualValue, setActualValue] = useState<IPerson>(
+    {name: '', surname:'', id: ''}
+  );
+  const [prevValue, setPrevValue] = useState<IPerson>(
+    {name: '', surname: '', id: ''}
+  );
+  const [activeEdit, setActiveEdit] = useState<boolean>(false);
+
+
+  const handleOpenNewMember = () => {
+    setOpenNewMember(true);
+  };
+
+  const handleCloseNewMember = () => {
+    setOpenNewMember(false);
+  };
+
+  const handleAcceptChange = (id: string) => {
+    rows && rows.map(el => {
+      if (el.id === id && (prevValue.name !== actualValue.name || prevValue.surname !== actualValue.surname )) {
+        editTeamMember(team, actualValue);
+
+        setPrevValue((prev) => (
+          {
+            ...prev,
+            name: actualValue.name,
+            surname: actualValue.surname
+          }
+        ));
+      }
+      return el;
+    });
+    onToggleEditMode(id);
+  };
+
+  const onToggleEditMode = (id: string) => {
+    if (activeEdit && rows && prevValue.id !== id) {
+      alert(`Jesteś w trakcie edycji innej osoby.`);
+      return;
+    };
+    setActiveEdit(!activeEdit);
+    rows && rows.find(el => {
+      if (el.id === id) {
+        setActualValue(el);
+      };
+    });
+    rows && setRows(() => {
+      return rows.map((row) => { 
+        if (row.id === id) {
+          setPrevValue(row);
+          return { ...row, isEditMode: !row.isEditMode };
+        }
+        return row;
+      });
+    });
+  };
+
+  const onChange = (e: { target: { getAttribute: (arg0: string) => string; value: any; name: string; }; }, row: { id: string; }) => {
+    if (e.target.getAttribute('name') === 'lp') return;
+    if (!previous[row.id]) {
+      setPrevious((state: any) => ({ ...state, [row.id]: row }));
+    }
+    const value = e.target.value;
+    const name = e.target.name;
+    const { id } = row;
+    if (rows) {
+      const newRows = rows.map(row => {
+        if (row.id === id) {
+          setActualValue((prev: any) => {
+            return { ...prev, [name]: value };
+          });
+          return { ...row, [name]: value };
+        }
+        return row;
+      });
+      setRows(newRows);
+    }
+
+  };
+
+  const onRevert = (id: string) => {
+
+    if (rows) { // if
+      const newRows = rows.map((row ) => {
+        if (row.id === id) {
+          setActualValue(prevValue);
+          return {
+            ...row,
+            name: prevValue.name,
+            surname: prevValue.surname
+          };
+        }
+        return row;
+      });
+      setRows(newRows);
+    }
+  };
+
+  const handleDelete = (id: string) => {
```

Comment:
At this time:  
1\. Comment on this method (for future uses).  
2\. On delete we should save dateOfDelete - for counting fee.

In next prototype we can think about this case.